Formula/bacula-fd.rb

```diff
@@ -18,24 +18,19 @@ def install
     #   (conio support not tested)
     # * working directory in /var/lib/bacula, reasonable place that
     #   matches Debian's location.
-    readline = Formula["readline"].opt_prefix
     system "./configure", "--prefix=#{prefix}",
-                          "--sbindir=#{bin}",
                           "--with-working-dir=#{var}/lib/bacula",
-                          "--with-pid-dir=#{HOMEBREW_PREFIX}/var/run",
+                          "--with-pid-dir=#{var}",
                           "--with-logdir=#{var}/log/bacula",
                           "--enable-client-only",
                           "--disable-conio",
-                          "--with-readline=#{readline}"
+                          "--with-readline=#{Formula["readline"].opt_prefix}"
 
     system "make"
     system "make", "install"
 
-    # Ensure var/run exists:
-    (var + "run").mkpath
-
-    # Create the working directory:
-    (var + "lib/bacula").mkpath
+    (var/"run").mkpath
```

Comment:
At this point, no. I remembered it once, but I didn't retain the reasoning. You'll have to ask @MikeMcQuaid.